CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 ### Fixed
 * <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
+* Slab usage was reported wrong ([#3284](https://github.com/realm/realm-core/pull/3284)
```

Comment:
When using which function was the output incorrect?

Reply:
better now?